html/pfappserver/lib/pfappserver/Form/Config/Source/Htpasswd.pm

```diff
@@ -36,6 +36,32 @@ has_field 'stripped_user_name' =>
              help => 'Use stripped username returned by RADIUS to test the following rules.' },
   );
 
+has_field 'realm' =>
```

Comment:
remove this once its in pfappserver::Form::Config::Source

Reply:
It's just common to internal source, so can it be define in pfappserver::Form::Config::Source ?